ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -721,9 +793,13 @@ class NGraphEncapsulateOp : public OpKernel {
         continue;
       }
 #endif
-      current_ng_tensor = GetCurrentNgTensor(
-          current_dst_ptr, last_dst_ptr, last_ng_tensor, true, ng_exec,
-          op_backend, ng_element_type, ng_shape);
+      if (m_executable_can_create_tensor) {
+        current_ng_tensor = out_group_from_pipeline[i];
```

Comment:
For output tensors, set_stale is always set to true 
https://github.com/tensorflow/ngraph-bridge/blob/951ca465be49bc271ae3b6e130d6d88a0324809f/ngraph_bridge/ngraph_encapsulate_op.cc#L1037